src/internal/item-container/index.tsx

```diff
@@ -292,10 +297,10 @@ function ItemContainerComponent(
   }
 
   function onBlur() {
-    // When drag- or resize handle loses focus the transition must be submitted with two exceptions:
+    // When drag- or resize handle on palette or board item loses focus the transition must be submitted with two exceptions:
     // 1. If the last interaction is not "keyboard" (the user clicked on another handle issuing a new transition);
-    // 2. If the item is borrowed (in that case the focus moves to the acquired item which is expected).
-    if (transition && transition.interactionType === "keyboard" && !transition.isBorrowed) {
+    // 2. If the item is borrowed (in that case the focus moves to the acquired item which is expected, borrowed item is hidden and all events on it should be muted).
```

Comment:
The "borrowed item" terminology might not be well understood now as there is no isBorrowed state anymore. I'd say that when the item is acquired by the board the palette item is hidden and muted - the event handlers must not fire for it.